test/test_helper.rb

```diff
@@ -0,0 +1,2 @@
+require 'simplecov'
+SimpleCov.start
```

Comment:
I think this was part of changing the file name for helper_test.rb to test_helper.rb

test_helper should include:
require 'simplecov'
SimpleCov.start
require 'minitest/autorun'
require 'minitest/pride'

We'll have to update the file requirement at the top of every test to reflect this name change